_pages/how-we-work/tools/murally.md

```diff
@@ -14,4 +14,9 @@ Because Mural is a web application, there’s no installation necessary. You wil
 
 - Mural consists of “rooms” (for example, FEC) and “murals” (for example, User Research). There’s only one “team:” 18F.
 - Rooms can be public or private to the rest of the team. If rooms within the 18F group are set to public, anyone at 18F can view them. If they’re private, only people specifically invited to them can see them.
-- You can give access to individual boards or rooms to anyone, including people outside of 18F or GSA, and doing so will not give them access to other public rooms without an administrator’s approval.
+- People outside of 18F or GSA can be added as guests to the 18F workspace. Guests have fewer permissions than members. They cannot create rooms, publish templates, or see open rooms. Guests do not cost any money to add, so follow the instructions in the next bullet to add your partners! 
```

Comment:
Still a bit confused about what guests *can* do if invited to the 18F workspace. Do they still need to be invited to see particular boards or rooms, or does inviting them as a guest eliminate the need to do that?

Reply:
Basically...I'm trying to figure out how to get a partner invited in such a way that you don't need to invite them to particular boards, but can instead just invite them once and they'd have access to all boards related to the project. Is the best process for that: 1) ask for partner to be invited as 18F guest in #admins-mural, 2) manually invite them to a room, 3) create all relevant boards in that room?

Reply:
I just emailed Mural to get the best answer. Will update when I hear!

Reply:
@hannahkane Check this out! Also made changes to the how to add guests part - I think it will be easier for people to add guests themselves.